### PR TITLE
[AIML-ADC-7313] Upgrade frontend dependencies

### PR DESCRIPTION
**Pull request by AlejandroRigau**
*Issue #, if available:* AIML-ADC-7313

*Description of changes:*
- Update all minor and patch changes
- add overrides to react-scripts dependencies
- add override to react-quill to use pre-release version of quill which fixes vulnerability.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -6,24 +6,24 @@
...
+        "@cloudscape-design/global-styles": "^1.0.25",
         "@reduxjs/toolkit": "^1.9.5",
-        "axios": "^0.27.2",
+        "axios": "^0.28.0",
         "git-repo-info": "^2.1.1",
-        "jest-mock-axios": "^4.7.0-beta",
+        "jest-mock-axios": "^4.7.3",
         "lodash": "4.17.21",
-        "quill": "^1.3.5",
+        "quill": "^2.0.0-rc.3",
```
Comment (dustins) [inline frontend/package.json:18]: Did you test the WYSIWYG editor for the create Groundtruth job page still works after this? Seems like 1.3.5 -> 2.0.0 is a big jump.

Comment (AlejandroRigau) [reply frontend/package.json:18]: image

Turns out this does break it. I need to figure out how to fix this

Comment (AlejandroRigau) [reply frontend/package.json:18]: As discussed in internal channel, I will revert this change and create a backlog item for updating quill to v2

**Comment by AlejandroRigau** [comment]
image
Back to normal functionality after reverting quill V2 update

**Review by KyleLilly — APPROVED** [review]

**Review by dustins — APPROVED** [review]

**Pull request merged**